apprise/plugins/NotifyGoogleChat.py

```diff
@@ -302,7 +353,7 @@ def parse_native_url(url):
 
         result = re.match(
             r'^https://chat\.googleapis\.com/v1/spaces/'
-            r'(?P<workspace>[A-Z0-9_-]+)/messages/*(?P<params>.+)$',
+            r'(?P<workspace>[a-zA-Z0-9_-]+)/messages/*(?P<params>.+)$',
```

Comment:
workspace names can include lower case letters

Reply:
This isn't necessary as the regular expression argument `re.I` (same as `re.IGNORECASE`) is passed into the `match()`

Reply:
I learnt ! It's fixed ;)